controller/src/beerocks/master/wfa_ca.h

```diff
@@ -40,8 +42,15 @@ class wfa_ca {
     static void reply_invalid_error(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx,
                                     eWfaCaStatus status, const std::string &description);
 
+    static bool reply_running(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx);
```

Comment:
Let me tell you that I really like how you introduced the basic wfa_ca class in the previous PR, and refactor new functionality into it every time you can use it. IMO that is much more useful than trying to think of all the needed functionality up front.

Reply:
thanks :)